ARCHITECTURE.md

```diff
@@ -0,0 +1,60 @@
+# Architecture
+
+This markdown file documents the architecture for the whole of [Scribe](https://github.com/scribe-org) - including the applications, the services, the processes, and the external systems and sources with which it interacts. As the file is meant to be a living document, edits are welcome to expand and update it!
+
+<a id="contents"></a>
+
+# **Contents**
```

Comment:
I'm going to make this an H2 and remove the bolding :) I realized just now for the docs that it's looking weird if we have this bolded. Having this included in the Scribe-Data docs would be really cool. Maybe we can do a hook to it somehow 🤔

Reply:
I think that back in the day when I was first doing readmes it was `**Contents**`, and then the bolding just stuck when it shifted to an H1.